src/components/vm/vmMigrateDialog.jsx

```diff
@@ -0,0 +1,206 @@
+    return (
+        <FormGroup label={_("Options")}
+                   hasNoPaddingTop>
+            <Checkbox id="live"
+                      isChecked={live}
+                      label={_("Run on destination")}
+                      onChange={b => {
+                          setLive(b);
+                          if (!b)
+                              setStorage("nocopy");
+                      }} />
+            {live && <Checkbox id="temporary"
+                               isChecked={temporary}
+                               label={
+                                   <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
+                                       <span>{_("Move temporarily")}</span>
+                                       <Tooltip position="top"
+                                                content={
+                                                    <>
+                                                        <p>{_("By default, the migrated VM config is removed from the source host, and saved persistently on the destination host. The destination host is considered the new home of the VM.")}</p>
+                                                        <p>{_("If 'temporary' is selected, the migration is considered only a temporary move: the source host maintains a copy of the VM config, and the running copy moved to the destination is only transient, and will disappear when it is shutdown.")}</p>
+                                                    </>}>
+                                           <OutlinedQuestionCircleIcon />
+                                       </Tooltip>
+                                   </Flex>
+                               }
+                               onChange={setTemporary} />
+            }
+        </FormGroup>
+    );
+};
+
+const StorageRow = ({ storage, setStorage, live }) => {
+    const copyDisabled = !live;
+    let copyRadio = (
+        <Checkbox id="copy"
+                  label={
+                      <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
+                          <span>{_("Copy storage")}</span>
+                          <Tooltip id="storage-copy-tooltip"
+                                   content={_("Migrate full disk images in addition to domain's memory. Only non-shared non-readonly disk images will be transferred.")}>
+                              <OutlinedQuestionCircleIcon />
+                          </Tooltip>
+                      </Flex>
+                  }
+                  isDisabled={copyDisabled}
+                  isChecked={storage === "copy"}
+                  onChange={isChecked => setStorage(isChecked ? "copy" : "nocopy")} />
+    );
+
+    if (copyDisabled) {
+        copyRadio = (
+            <Tooltip id="storage-copy-tooltip"
+                     content={_("Offline migration doesn't allow copying storage")}>
```

Comment:
This is very counter-intuitive (as above), but maybe some weird technical limitation in libvirt. Can we do this without exposing the storage option at all, and avoid all this confusion?

If you need to keep this, please s/doesn't/does not/. Contractions are fine for casual/spoken English, but we shouldn't (haha) have them in the UI.